client/src/ui/molecules/search/index.tsx

```diff
@@ -22,7 +22,7 @@ export function Search(props) {
   }, []);
 
   return (
-    <div className="header-search">
+    <div className="header-search" role="search">
```

Comment:
I think the `<form>` part of BasicSearchWidget might be better https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role#prefer_html

Reply:
That was my first inclination as well but then I remember reading(in the ARIA docs I believe) that it is not recommended to add it directly to the `form` element. But then 🙃 I found this thread upon further investigation into the matter:

https://github.com/w3c/wcag/issues/113

It, therefore, seems that adding the `role` to this form in this instance is allowed and would be the preferred method. With regards to the search results page, I reckon there adding it to the container that will also include the search results is going to be the appropriate option.

Thoughts @nschonni? - Thank you for your comment, it got me to dig deeper and find the above thread.